service/core.go

```diff
@@ -49,6 +49,12 @@ type Config struct {
 	RootContent           io.ReadSeeker
 	Version               string
 	Logger                *zap.Logger
+	// EnableStayAlive when true enables the stayalive endpoint. If a client
+	// begins polling the stayalive endpoint then exits, the Zed process will
+	// also exit. This is used by the Zui application to ensure that the Zed
+	// process exits in the event that the Zui process is terminated with
+	// SIGKILL.
+	EnableStayAlive bool
```

Comment:
For parity with -brimfd, I think we want this to be something like `StayAliveTimeout time.Duration` that, if greater than zero, both enables the /stayalive route and starts a timer that will kill the server on expiration if it hasn't received a /stayalive request.

Reply:
Done.